projects/plugins/boost/app/lib/Setup.php

```diff
@@ -27,7 +27,7 @@ class Setup {
 	 * @return void
 	 */
 	public static function add( Has_Setup $instance ) {
-		self::$instances[] = $instance->do_setup();
+		self::$instances[] = $instance->setup();
```

Comment:
`setup` may not be returning the instance.

```suggestion
    $instance->setup()
    self::$instances[] = $instance;
```

Reply:
👀 [884e1c8](https://github.com/Automattic/jetpack/pull/28888/commits/884e1c8403d268d499bb551ca8d54af8be6a01db)